images/loading-thumb.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="20" height="20" viewBox="0 0 20 20">
```

Comment:
From now on, please optimize the SVGs you add using [svgo](https://github.com/svg/svgo). It's really easy to do.
```
# install globally so you have it available
npm install -g svgo
# run on the whole image dir or the file you added specifically
svgo images/
svgo images/loading-thumb.svg
```